src/main/webapp/js/viewtrails.js

```diff
@@ -9,7 +10,31 @@ $(window).load(function(){
     };
 
     map = new google.maps.Map(document.getElementById('map'), mapOptions);
+    map.addListener('dragend', function() {
+        var latlng = map.getCenter().toJSON();
+        console.log(latlng);
+        geocoder.geocode({'location': latlng}, function(results, status) {
+          if (status === 'OK') {
+            if (results[0]) {
+              var currentLocationElement = document.getElementById('current-location');
+              var country = ""
+              results[0].address_components.forEach((component) => {
+                if (component.types.toString().includes('country')){
+                  country = component.long_name;
+                }
+              });
+              currentLocationElement.innerHTML = "<h1>"+country+"</h1>";
```

Comment:
It's very convenient to use backtick to format a string.
For example `"<h1>"+country+"</h1>"` can be written as `` `<h1>${country}</h1>` ``
So you don't have to write some "+"s

Reply:
That's neat. But, this text format will change in future. This one is for testing.